gitea/files/locale/custom/locale_en-US.ini

```diff
@@ -0,0 +1,10 @@
+sign_up = Register a new user in Altinn Studio
+
+[startpage]
+app_desc = Git for your apps
+
+[auth]
+login_userpass = Log in to Altinn Studio
+oauth_signup_title = Register your email and create a password for Altinn Studio. This is used, for example, to recover your account.
+authorize_application_description = If you do, this API can read and write to all your account information, including private repos and organizations.
```

Comment:
No, this means that Altinn Studio can read private repos and organizations from Gitea.
I realize that this information is only in the title:
`authorize_title = Do you want to give "%s" access to your Gitea account?`

<img width="474" alt="confirmation-dialog" src="https://github.com/user-attachments/assets/02c66dfb-06e5-4ff1-962b-39a50fad0936">

So we should probably specify it in the text as well 🤔
I also wonder if we shouldn't rather refer to an `application` instead of an `API` 🤔.

@Ildest @ErlingHauan What do you think about updating the text with something like:
`If you do, this application can read and write to all your Gitea account information, including Gitea private repos and organizations.`

Reply:
@mlqn I see! Then I think it makes sense as it is 😊
But I agree on changing API to Application 👍

Reply:
@mlqn If it is the application you authorise, then yes, write that. Remember to write "authorise" (I see two instances of it in the message above) _without a capital letter_ and exchange z with s (because we use British English i Digdir). The same with organization - with an s instead of z.

Reply:
I am a bit confused as to whether this message will be in English or Norwegian now though :-D.

Reply:
Regarding British English: I see that the language file is called `locale_en-US`. Maybe there exists a `locale_en-GB` file we could use instead? But maybe that is material for a second PR 😊

Reply:
Ok, thanks for the feedback! :)
The message will be available in both Norwegian and American English. Gitea doesn't seem to have a British English version available. If we want one, we will need to create it in Crowdin and fix the 3600 sentences 🫤